common/buildings/00_tradepost.txt

```diff
@@ -463,3 +463,294 @@ trade_post = {
 		# extra_tech_building_start = 10.0 # Never get at start of game
 	# }
 }
+
+##########################################
+#
+# Ciphers Trade Posts mod merger buildings
+#
+##########################################
+
+trade_post = {	
+	
+#Garrison
+
+	#StableMaster
+	wc_stableMaster = {
+		 desc = wc_stableMaster1_desc
+		 potential = {
+			always = yes
+		 }
+		 gold_cost = 650
+		 build_time = 365
+		 local_tax_modifier = 0.1
+		 trade_route_wealth = 0.5
+		 ai_creation_factor = 100
+	}
+	
+	#FlightMaster
+	wc_flightMaster = {
+		 desc = wc_flightMaster_desc
+		 potential = {
+		 	always = yes
+		 }
+		 upgrades_from = wc_stableMaster
+		 gold_cost = 900
+		 build_time = 365
+		 local_tax_modifier = 0.2
+		 trade_route_wealth = 1
+		 ai_creation_factor = 100
+	}
+	
+	#WorgBestiary
+	#wc_worgBestiary = {
+	#	 desc = wc_worgBestiary_desc
+	#	 potential = {
+	#		always = yes
+	#		owner = { culture_group = orc_group }
+	#	 }
+	#	 gold_cost = 650
+	#	 build_time = 365
+	#	 local_tax_modifier = 0.1
+	#	 local_movement_speed = 0.2
+	#	 trade_route_wealth = 0.5
+	#	 ai_creation_factor = 100
+	#}
+	
+	#Wyvern Master
+	#wc_wyvernMaster = {
+	#	 desc = wc_wyvernMaster_desc
+	#	 potential = {
+	#		always = yes
+	#		owner = { culture_group = orc_group }
+	#	 }
+	#	 
+	#	 upgrades_from = wc_worgBestiary
+	#	 gold_cost = 900
+	#	 build_time = 365
+	#	 local_tax_modifier = 0.2
+	#	 local_movement_speed = 0.5
+	#	 trade_route_wealth = 1
+	#
+	#	ai_creation_factor = 100
+	#}
+
+	#Bank
+	wc_bank	 = {
```

Comment:
This building should not be in non-trade route posts. `trade_route_wealth` isn't used in republic trade posts.